gobblin-metrics/src/main/java/gobblin/metrics/MetricContext.java

```diff
@@ -117,6 +139,13 @@ private MetricContext(String name, MetricContext parent, List<Tag<?>> tags,
     }
   }
 
+  private ExecutorService getExecutorService() {
```

Comment:
This method should be `synchronized` in case `sendEvent` is called from multiple threads.